charts/mattermost-enterprise-edition/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Mattermost Enterprise server with high availitibity.
 name: mattermost-enterprise-edition
-version: 1.8.0
+version: 1.8.1
```

this is new features, can you bump to
```suggestion
version: 1.9.0
```

---

commited to the latest commit